tasks/post_to_database/post_to_database.py

```diff
@@ -10,11 +10,13 @@
 # noinspection SpellCheckingInspection
 import fastjsonschema as fastjsonschema
 from cumulus_logger import CumulusLogger
+from orca_shared import shared_recovery
```

Comment:
Any reason this is being imported 2x on line 13 and 19? 
```
from orca_shared import shared_recovery
from orca_shared.recovery import shared_recovery
```

Reply:
Seems not. My environment is weird about imports sometimes. Removed.